src/helpers/mailHelper.js

```diff
@@ -0,0 +1,9 @@
+import nodemailer from 'nodemailer';
+
+export const transporter = nodemailer.createTransport({
+  service: 'gmail',
+  auth: {
+    user: 'muhire416@gmail.com',
```

Comment:
create a new gmail account that's specifically for barefoot-nomad-phoenix. And put the credentials in the .env